### PR TITLE
adding table styling

### PR DESCRIPTION
**Pull request by Anand-Issac**
adding some table styling to _config.scss

### CODE REVIEW

File: client/src/style/_config.scss [hunk 1]
```diff
@@ -113,3 +113,25 @@ $button-disabled-background: #b2b2b2;
 
 /* Tag Config */
 $tag-background: #ededed;
+
+/* Table Config */
+/* Table Config */
```
Comment (jlight99) [inline client/src/style/_config.scss]: nit: remove duplicate `/* Table Config */`

File: client/src/style/_config.scss [hunk 2]
```diff
@@ -113,3 +113,25 @@ $button-disabled-background: #b2b2b2;
...
+@mixin table-base {
+    font-family: "Inter";
+    margin-top: 20px;
+    .a{
+        color: #333333;
```
Comment (jlight99) [inline client/src/style/_config.scss:125]: nit: assign #333333 and #969696 to global scss variables in _config.scss

File: client/src/style/_config.scss [hunk 3]
```diff
@@ -113,3 +113,25 @@ $button-disabled-background: #b2b2b2;
 
 /* Tag Config */
 $tag-background: #ededed;
+
+/* Table Config */
+/* Table Config */
+@mixin table-base {
+    font-family: "Inter";
+    margin-top: 20px;
+    .a{
```
Comment (jlight99) [inline client/src/style/_config.scss:124]: nit: add space (`.a {`)

File: client/src/style/_config.scss [hunk 4]
```diff
@@ -113,3 +113,25 @@ $button-disabled-background: #b2b2b2;
...
+        color: #333333;
+    }
+    & td {
+      border-top: unset;
+    }
+    .row-style{
```
Comment (jlight99) [inline client/src/style/_config.scss:130]: nit: add space: `.row-style {`

File: client/src/style/_config.scss [hunk 5]
```diff
@@ -113,3 +113,25 @@ $button-disabled-background: #b2b2b2;
...
+        font-size: 16px;
+        text-align: center;
+    }
+    .header-style{
```
Comment (jlight99) [inline client/src/style/_config.scss:134]: nit: add space: `.header-style {`

**Review by jlight99 — APPROVED** [review]
just some formatting changes, but looks good! btw do you have any examples/screenshots of using this styling on a table?

**Review by meganniu — APPROVED** [review]
LGTM!!

**Review by guy-ro — APPROVED** [review]
LGTM

**Pull request merged**